openapi.yaml

```diff
@@ -10692,6 +10692,22 @@ paths:
                   type: string
                   description: The label for this keypair, for display purposes only.
                   example: my-key
+                bucket_access:
+                  type: object
```

Comment:
This should be `type: array` with `items` being of `type: object` and having the properties you added below. Same for the response value below.